.github/workflows/cluster_endtoend_12.yml

```diff
@@ -102,7 +102,7 @@ jobs:
         go install github.com/vitessio/go-junit-report@HEAD
 
     - name: Setup launchable dependencies
-      if: steps.skip-workflow.outputs.skip-workflow == 'false' && steps.changes.outputs.end_to_end == 'true'
+      if: steps.skip-workflow.outputs.skip-workflow == 'false' && steps.changes.outputs.end_to_end == 'true' && github.base_ref == 'main'
```

Comment:
The `github.base_ref` variable is only available in workflows triggered by Pull Requests. Meaning that the Launchable integration will not run on `main`. Is this intentional?

Reply:
Yes, I saw this. Not sure if this is super important or not. WDYT?

Reply:
I think it would be fine to skip Launchable on `main`. The data collected from the PRs is already interesting.